drivers/spi/spi_ll_stm32.c

```diff
@@ -178,22 +179,22 @@ static void spi_stm32_complete(struct spi_stm32_data *data, SPI_TypeDef *spi,
 			       int status)
 {
 #ifdef CONFIG_SPI_STM32_INTERRUPT
-	LL_SPI_DisableIT_TXE(spi);
-	LL_SPI_DisableIT_RXNE(spi);
-	LL_SPI_DisableIT_ERR(spi);
+	spi_stm32_ll_func_DisableIT_TXE(spi);
```

Comment:
Can we group these under `spi_stm32_ll_func_disable_it` ?

Reply:
Since we need to keep ll_func_enable_IT_TXE(spi), ll_func_enable_IT_RXNE(spi), ll_func_enable_IT_ERR(spi), ll_func_disable_IT_TXE(spi), ll_func_disable_IT_RXNE(spi), I thought that keeping ll_func_disable_IT_ERR(spi) instead of grouping those three lines was better for consistency sake